Okta/OktaRevoke.swift

```diff
@@ -9,6 +9,7 @@
  *
  * See the License for the specific language governing permissions and limitations under the License.
  */
+import Hydra
```

Comment:
We can remove this import, since it isn't used in this file.

Reply:
Done.